### PR TITLE
Error Catching and just some fixes here and there (like adding streamlabs link to funding.yml and typos!)

### PR DESCRIPTION
**Pull request by OkiStuff**
I made the error catching which you can see in coronapy/libs/error.py. also fixed some typos. added requests to the setup.py. and thats really it. not much. Hope it looks good

### CODE REVIEW

**Comment by OkiStuff** [comment]
also added streamlabs donation page to funding.yml

**Comment by OkiStuff** [comment]
Funding.yml now works with two links

File: coronapy/lib/error.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+def crash_msg(error_code, file: str, fixes: str):
+
+    pecho( color.prCyan(":(") )
+    pecho(color.prRed("There was an error which forced Coronapy to crash") )
+    pecho(color.prGreen("But you can report this bug! https://github.com/MouadBH/coronapy-cli/issues") )
+    pecho(color.prPurple("Before you report the bug, please try these fixes: " + fixes) )
+    pecho(color.prRed("ERROR Codesfdsfdasfsa: \n" + str(error_code) + "in file " + file + "\n You can put this error code in the bug report (it would help us out)") )
+
```
Comment (MouadBH) [inline coronapy/lib/error.py:23]: You only have to add `exit()` here to avoid printing python error message.

**Comment by MouadBH** [comment]
Thank yo so mush, You did a lot of work in the project, I added some comments, check them out.

**Comment by OkiStuff** [comment]
Um @MouadBH where are the comments?

**Comment by MouadBH** [comment]
i forgot to submit the review sorry.

File: coronapy/lib/error.py (hunk 1 above)
Comment (OkiStuff) [reply coronapy/lib/error.py:23]: Added exit()

**Comment by OkiStuff** [comment]
Done.. When can you merge?

**Comment by OkiStuff** [comment]
Fixed some typos

**Comment by MouadBH** [comment]
That really great and helpful, thank you for your contribution and your hard work.

**Pull request merged**